Makefile

```diff
@@ -297,6 +303,12 @@ test-e2e:
 	cd deploy; source ./../e2e/local-e2e.sh && run $(driver)
 	$(GO) test -v $(shell $(GO) list ./e2e)
 
+.PHONY: actions-e2e
+actions-e2e:
+	cd deploy; source ./../e2e/ci-e2e.sh && run "virtualbox" $(VERSION)
+	#$(GO) test -v $(shell $(GO) list ./e2e)
+	#./e2e/e2e-dump.sh
```

Comment:
for the e2e dump to work we should put this into a separate github actions step, as make will stop executing the target if the e2e test run itself fails, and especially those situations we're interested in the logs